lua/entities/gmod_wire_egp/lib/egplib/objectcontrol.lua

```diff
@@ -179,7 +179,7 @@ function EGP:PerformReorder( Ent )
 	-- Second pass, remove objects from their original positions
 	for i=#Ent.RenderTable,1,-1 do
 		local obj = Ent.RenderTable[i]
-		if obj.ChangeOrder then
+		if obj and obj.ChangeOrder then
```

Comment:
This is just covering up a bug caused by something else. There should not be a nil partway in the RenderTable.